storagetransfer/storagetransfer_test.go

```diff
@@ -289,12 +290,14 @@ func TestTransferUsingManifest(t *testing.T) {
 	object := sc.Bucket(gcsSourceBucket).Object("manifest.csv")
 	defer object.Delete(context.Background())
 
-	resp, err := transferUsingManifest(buf, tc.ProjectID, sourceAgentPoolName, rootDirectory, gcsSinkBucket, gcsSourceBucket, "manifest.csv")
-	defer cleanupSTSJob(resp, tc.ProjectID)
+	testutil.Retry(t, 5, time.Second, func(r *testutil.R) {
+		resp, err := transferUsingManifest(buf, tc.ProjectID, sourceAgentPoolName, rootDirectory, gcsSinkBucket, gcsSourceBucket, "manifest.csv")
+		defer cleanupSTSJob(resp, tc.ProjectID)
 
-	if err != nil {
-		t.Errorf("transfer_using_manifest: %#v", err)
-	}
+		if err != nil {
+			t.Errorf("transfer_using_manifest: %#v", err)
```

Comment:
This needs to use `r` instead of `t`. Otherwise, the test will still fail even if it succeeds on retry.

Reply:
gotcha, that only makes sense :)